Umbraco-Cloud/Troubleshooting/Deployments/Changes-Not-Being-Applied/index.md

```diff
@@ -0,0 +1,53 @@
+---
+versionFrom: 7.0.0
+---
+
+# Extraction error: Config transforms failing
+
+Sometimes you might notice that even though your deployments are coming through without errors, no changes are actually being applied on the Cloud environments.
+This might be due to Config Transforms failing during the extraction process.
+
+This issue may occur if you have added custom [Config Transforms](../../../Set-up/Config-Transforms) to your project.
+
+Since Config Transforms are applied on every data extraction, the behaviour will continue until the Config Transform files are either corrected, or removed.
+
+## How do I know I have this issue?
+
+This problem is a bit tricky to spot, as the only indicator is that changes you made on the original environment are not being applied even though the deployment was complete (although when pushing from local you might get a warning message).
+
+The environment overview on your project will most likely not mention anything, which means you will have to delve into KUDU.
+
+## How to check for and resolve the issue
+
+1. Access [KUDU](../../../Set-up/Power-Tools)  on the affected environment
```

Comment:
How would the reader know which environment is the "affected environment"?
Would it perhaps be an idea to say something like "the environment where you expected to see your changes reflected"? 🤔